bundles/automation/org.eclipse.smarthome.automation.api/src/main/java/org/eclipse/smarthome/automation/parser/Converter.java

```diff
@@ -9,14 +9,21 @@
 
 import java.util.Dictionary;
 
+/**
+ * This interfase provides opportunity to use different parsers for converting JsonObject to Dictionary.
+ *
+ * @author Vasil Ilchev - Initial Contribution
+ *
+ */
 public interface Converter {
```

Comment:
Do we at all need this separate interface? Couldn't the method be moved into the Parser interface?